layouts/shortcodes/compat-table-ossmc.html

```diff
@@ -0,0 +1,24 @@
+{{ $data := index .Site.Data.compatibility.ossmc }}
+
+<table>
+  <thead>
+    <tr>
+      <th style="width: 100px">Openshift</th>
```

Comment:
sorry - missed this one :)
```suggestion
      <th style="width: 100px">OpenShift</th>
```